sql/src/main/antlr/OpenSearchSQLParser.g4

```diff
@@ -627,10 +635,19 @@ qualifiedName
 ident
     : DOT? ID
     | BACKTICK_QUOTE_ID
+    | metadataField
     | keywordsCanBeId
     | scalarFunctionName
     ;
 
+metadataField
+    : META_INDEX
+    | META_ID
+    | META_SCORE
+    | META_MAXSCORE
+    | META_SORT
+    ;
```

Comment:
It looks like `metadataField` and all `META_` tokens are no longer necessary.

Reply:
Interesting... yes.  I think I can remove these.